### DISCUSSION TITLE
Welcome to Avalon Discussions!

### DISCUSSION THREAD

**Discussion by mottosso** [getavalon/core#566] [General]
## 👋 Welcome!
  
Let's see how this works. :)

**Comment by davidlatwe** [comment]
wooooowww🥳🥳

**Comment by mkolar** [comment]
I've been digging through quite a few other repos that have been using it for a while and like it a lot. Feels a lot more organised than simple chat. Kinda like a focused stackoverflow.

**Comment by mottosso** [comment]
Just too bad there hasn't been any activity. Why not have the next conversation here?
